src/EntityComponentManager_TEST.cc

```diff
@@ -107,149 +107,24 @@ class EntityComponentManagerFixture : public ::testing::TestWithParam<int>
   public: EntityCompMgrTest manager;
 };
 
-/////////////////////////////////////////////////
-TEST_P(EntityComponentManagerFixture, AdjacentMemorySingleComponentType)
```

Comment:
I removed tests related to adjacent/sequential memory since the old `ComponentStorage` approach is no longer used.